app/misc/router.ts

```diff
@@ -311,6 +314,31 @@ function hideDiffPanelButtons() {
   disableDiffPanelEditOnHide();
 }
 
+function checkRepoOpen() {
+  // hide these repo nav elements if there is no repo
+
+  // this checks to see if a repo has successfully been open
+  let repoElement = document.getElementById("repo-name");
+  // this checks to see if the user set a repo path to open
+  let repoPath = document.getElementById("repoOpen");
+  let repoCreate = document.getElementById("repoCreate");
+  let repoClone = document.getElementById("repoClone");
+
+  let showRepoNavTools = "hidden";
+  // if these values are set, then show everything
+  if (repoElement.innerHTML != "repository" || repoPath.value ||
+      repoCreate.value || repoClone.value || repoLocalPath)
+  {
+    showRepoNavTools = "visible";
+  }
+
+  // show/hide the relevent items
+  document.getElementById("repo-back-button")!.style.visibility = showRepoNavTools;
+  document.getElementById("nav-repo-branch-tag-info")!.style.visibility = showRepoNavTools;
+  document.getElementById("nav-toolbar")!.style.visibility = showRepoNavTools;
+  document.getElementById("nav-open-repo-button")!.style.visibility = showRepoNavTools;
```

Comment:
Why'd you hide the folder button? I liked it being always visible

Reply:
You and @maxUser can duke it out... let me know the result.

Reply:
I concur with @mehrler12 but in addition to this if I do not have a repo open, there's no clear way of returning to the "open a repo" screen without the folder button. And as of right now (I know there's an issue to fix this) but visualgit always opens to that screen so now I can't open a repo at all.

Reply:
> I concur with @mehrler12 but in addition to this if I do not have a repo open, there's no clear way of returning to the "open a repo" screen without the folder button. And as of right now (I know there's an issue to fix this) but visualgit always opens to that screen so now I can't open a repo at all.
> This is the screen visualgit opens to, and I can do nothing now as I have no repo open and no way to open one?
> ![repofolder](https://user-images.githubusercontent.com/31779196/59942330-315dd300-9414-11e9-899e-6505035994e6.png)

We either have to abandon this whole change or live with it. Why does VG open to this screen is the question?

Reply:
Tested it with the new updates and I'm having the same problem as @hdauvic. VisualGit opens to the graph page by default, so I have no way of getting to the "open repo" page. I think you should either add the folder back or find a way to open VisualGit to the correct page.

Reply:
"Please try again." :)